### PR TITLE
Move PCL compatible helpers to AdvApi32.Shared project

### LINKED ISSUE #90: Move helpers compatible with PCL from AdvApi32.Desktop to AdvApi32 (Portable)
After this change #87 I think should be good to provide the portable project with the helper methods.

### PR DESCRIPTION
**Pull request by jmelosegui**
Fix #90

### CODE REVIEW

**Comment by AArnott** [comment]
@jmelosegui: please title your pull requests with meaningful titles, and avoid putting the bug ID you're fixing in the title as it makes it difficult to read. The title currently appears in GitHub as "Fix #90 #96", which isn't very informative.
Another thing to consider is that the PR title becomes the default merge commit message IIRC. So it's good to make sure it describes what the change is rather than referencing numbers that don't mean anything outside the context of your git history.

Mentioning the bug ID it fixes is great to include in the PR comments though.

File: src/AdvApi32/AdvApi32.csproj [hunk 1]
```diff
@@ -29,13 +29,16 @@
...
+      <Project>{1fdc092a-d3e8-4cc4-b896-17e0a213f723}</Project>
+      <Name>Kernel32</Name>
+    </ProjectReference>
     <ProjectReference Include="..\Windows.Core\Windows.Core.csproj">
       <Project>{b08c3c79-4cdd-4d37-933c-07d3452fd5f1}</Project>
       <Name>Windows.Core</Name>
     </ProjectReference>
   </ItemGroup>
   <Import Project="..\AdvApi32.Shared\AdvApi32.Shared.projitems" Label="Shared" />
-  <Import Project="..\Kernel32.Shared\Kernel32.Shared.projitems" Label="Shared" />
```
Comment (AArnott) [inline src/AdvApi32/AdvApi32.csproj:38]: Wow. I hadn't noticed this was there before. Thanks for removing it. Do you know of any other references to shared.projitems that appear elsewhere besides kernel32 projects?

**Comment by AArnott** [comment]
LGTM

File: src/AdvApi32/AdvApi32.csproj (hunk 1 above)
Comment (jmelosegui) [inline src/AdvApi32/AdvApi32.csproj:38]: No, I don't. And that was probably me (when I added that project). Sorry for that.

**Pull request merged**